source/whats-new.txt

```diff
@@ -0,0 +1,60 @@
+.. _ruby-whats-new:
+
+==========
+What's New
+==========
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: update, new feature, deprecation, upgrade
+
+Learn what's new in:
+
+* :ref:`2.21 <version-2.21>`
+* :ref:`2.20 <version-2.20>`
+
+.. _upcoming-breaking-changes:
+
+.. _version-2.21:
+
+What's New in 2.21
+------------------
+
+The {+driver-short+} 2.21 release includes the following new features: 
+
+- Supports the Client-Side Operations Timeout (CSOT) feature, which unifies
+  most timeout-related options under a single ``timeout_ms`` option.
+- Supports {+mdb-server+} version 8.0.
+- Support for range v2 queries with Queryable Encryption, including a new ``trim_factor``
+  parameter. For more information about Queryable Encryption, see :manual:`Queryable Encryption </core/queryable-encryption>`
+  in the {+mdb-server+} manual.
+
+.. _version-2.20:
+
+What's New in 2.20
+------------------
+
+The {+driver-short+} 2.20 release includes the following new features:
+
+- Discontinues support for Ruby 2.5 and 2.6. Deprecates support for Ruby 2.7 and
+  JRuby 9.2, which will be discontinued in the next minor driver version. Adds
+  support for JRuby 9.4.
+- Supports the newly-released Ruby-BSON version 5.0.
+- Allows connection strings without a slash between the hosts and the options.
+  For example, "mongodb://example.com?w=1" and "mongodb://example.com/?w=1" are
```

Comment:
```suggestion
  For example, ``mongodb://example.com?w=1`` and ``mongodb://example.com/?w=1`` are
```
Suggestion: I normally put connection strings in monospace. I don't see this in the style guide, but it's also a bit easier to read when it's distinguished from the rest of the text.

Reply:
I was going to do this as well! I copied this content over from the old release notes page so left as-is, but I agree with you and will update.